### ISSUE TITLE
GinkgoLogr does not produce newlines

### ISSUE THREAD

**Issue by erdnas** [onsi/ginkgo#1170]
```
GinkgoLogr.Info("Something")
GinkgoLogr.Info("Something else")
```

Outputs:
```
  Timeline >>
  "level"=0 "msg"="Something""level"=0 "msg"="Something else"
...
```

I would expect it to have a newline after each log call, for readability.
```
  Timeline >>
  "level"=0 "msg"="Something"
  "level"=0 "msg"="Something else"
...
```

If we add a newline to the printf here (or change it to println), we get newlines as desired.
https://github.com/onsi/ginkgo/blob/9b5078f221b4d21085cf45e7e8a0fca03c2a8466/internal/writer.go#L136-L140

**Comment by onsi** [comment]
hey @Nuckal777 can you take a quick look at this.  i know you implemented the logr stuff back in #1067 - is the newline being proposed here consistent with the logr spec?

**Comment by Nuckal777** [comment]
Good catch, we should a `\n` to the `printf`.
